### PR TITLE
Added pypy 7.3.13

### PR DESCRIPTION
**Pull request by dand-oss**
Added pypy 7.3.13

### CODE REVIEW

**Comment by native-api** [comment]
Please take a look at the CI failures.

**Comment by dand-oss** [comment]
> Please take a look at the CI failures.

Add PyPy 7.3.10 #2553
https://github.com/pyenv/pyenv/pull/2553

l see, the 2.7 error we gave up on here was not pip...

/Users/runner/work/pyenv/pyenv/versions/pypy2.7-7.3.13/bin/python: No module named pip (to install pip, you need to run once "/Users/runner/work/pyenv/pyenv/versions/pypy2.7-7.3.13/bin/python -m ensurepip")

File: .github/workflows/modified_scripts_build.yml [hunk 1]
```diff
@@ -58,6 +58,7 @@ jobs:
             python -c 'import sys; print(sys.version)'
           else
             python --version
+            python -m ensurepip
```
Comment (native-api) [inline .github/workflows/modified_scripts_build.yml:61]: Pyenv is supposed to install Pip in the course of `pyenv install`. If it doesn't something is wrong.

**Review by native-api — CHANGES_REQUESTED** [review]
Something goes wrong in `build_package_ensurepip()`.

I'll try to reproduce it in my MacOS VM.

File: .github/workflows/modified_scripts_build.yml (hunk 1 above)
Comment (dand-oss) [reply .github/workflows/modified_scripts_build.yml:61]: check...  bash function not succsful on pypy 2.7 mac

**Review by native-api — APPROVED** [review]

**Pull request merged**